src/data/resolvers/token.ts

```diff
@@ -201,5 +207,58 @@ export const tokenResolvers = ({
         colonyAddress,
       }));
     },
+    async processedBalances(
+      { address }: { address: Address },
+      { colonyAddress }: { colonyAddress: Address },
+      { client }: { client: ApolloClient<object> },
+    ) {
+      try {
+        const { data } = await client.query<
+          SubgraphDomainsQuery,
+          SubgraphDomainsQueryVariables
+        >({
+          query: SubgraphDomainsDocument,
+          variables: {
+            /*
+             * Subgraph addresses are not checksummed
+             */
+            colonyAddress: colonyAddress.toLowerCase(),
+          },
+          fetchPolicy: 'network-only',
+        });
+
+        if (data?.domains) {
+          const domains = [...data.domains, { domainChainId: '0' }];
+
+          const colonyClient = await colonyManager.getClient(
+            ClientType.ColonyClient,
+            colonyAddress,
+          );
+
+          const balances: BigNumber[] = await Promise.all(
+            domains.map((domain) =>
+              getBalanceForTokenAndDomain(
+                colonyClient,
+                address,
+                Number(domain.domainChainId),
+              ),
+            ),
+          );
+
+          return domains.map((domain, idx) => ({
+            __typename: 'DomainBalance',
+            amount: balances[idx].toString(),
+            domainId: Number(domain.domainChainId),
+            // `address` & `colonyAddress` only used for cache key - NOT PART OF QUERY
```

Comment:
I don't really think this is true in this case, but frankly, I don't really care right now :)